edx_notifications/server/web/static/edx_notifications/js/views/notification_pane_view.js

```diff
@@ -54,7 +58,21 @@ var NotificationPaneView = Backbone.View.extend({
 
         this.hydrate();
     },
-
+    enableMcKi18nNamespace: function(){
+      var i18nNamespace = 'mcka_apros';
+      if(i18nNamespace in window && window[i18nNamespace].gettext){
+        i18nNamespace = window[i18nNamespace];
+
+        window.pluralidx = i18nNamespace.pluralidx;
```

Comment:
@naeem91 is there a reason to make these variables global by putting into window object? Can we use some better approach and avoid this one?

Reply:
Otherwise we'll have to prefix all i18n methods in the code with the apros namespace. i.e; `mcka_apros.gettext` etc. Overriding them at one place is helpful in easy updates e.g; in case we want to remove or change namespace.

Reply:
I am just curious what would be the drawbacks of this approach?

Reply:
what if you create a utility function and reuse that function everywhere? and what if we define the namespace variable in settings and use it?

Reply:
> what if you create a utility function and reuse that function everywhere? and what if we define the namespace variable in settings and use it?

yes, that's possible